app/components/transfer/TransferSummaryPage.js

```diff
@@ -176,7 +176,7 @@ export default class TransferSummaryPage extends Component<Props> {
                       selectedExplorer={this.props.selectedExplorer}
                       light
                       hash={this.props.addressToDisplayString(deregistration.rewardAddress)}
-                      linkType="address"
+                      linkType="stakeAddress"
```

Comment:
should we maybe have it default to `address` if `stakeAddress` isn't set for an explorer

Reply:
Hmm thinking more about this, I'm not sure if we should have this kind of fallback. Instead, @ashisherc can you please add the `stakeAddress` for the other explorers? CardanoScan can't be the only one that supports stake addresses